source/shared/utils/short-address.js

```diff
@@ -10,6 +10,7 @@ const defaultConfig = {
 const shortAddress = (address, config = defaultConfig) => {
   if (!address) return '';
   if (typeof address !== 'string') return '';
+  if (address.substr(address.length - 4) == '.icp') return address;
```

Comment:
Would be nice to abstract this to a constant or build a utils function to handle these icns type of validations